### PR TITLE
Feature/remove yui frontend

### PR DESCRIPTION
**Pull request by rushton3179**

### CODE REVIEW

File: docs/INSTALL.md [hunk 1]
```diff
@@ -96,19 +96,42 @@ docker exec -it <mongo-container-name> bash
...
+
+# start the mongo command line
+mongo
+```
+```js
+// select and create the admin db
 use admin
 
-# update the auth schema
+// update the auth schema
```
Comment (kevinoshea) [inline docs/INSTALL.md]: Why is this changed?

Comment (rushton3179) [reply docs/INSTALL.md]: Its due to mongo having different comment identifiers, this allows copy paste of the commands when setting up
